command/harness/setup.go

```diff
@@ -245,11 +245,8 @@ func handleSetup(
 	stageRuntimeID := r.ID
 
 	// try to provision an instance from the pool manager.
-	var query *types.QueryParams
-	if poolManager.IsDistributed() {
-		query = &types.QueryParams{
-			RunnerName: runnerName,
-		}
```

Comment:
you tested for non HA builds as well ?

Reply:
non ha builds will have same runnerName in all instances, so query won't matter.